rnndb/adreno/adreno_pipe_regs.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<database xmlns="http://nouveau.freedesktop.org/"
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
+
+<!--
+	Pipe registers are a special kind of registers used by SQE on a6xxx, and
+	on a5xx by the ME, to control the CP. They only exist inside the CP, can
+	only be written to, and use the high 8 bits of the $addr register. For
+	example, this is how CP_WAIT_MEM_WRITES is implemented on a6xx:
+
+CP_WAIT_MEM_WRITES:
+	053b: 8b1d0084	mov $addr, 0x0084 << 24
```

Comment:
btw, might be useful to support this in afuc, ie. something like
```
  mov $addr, @WAIT_MEM_WRITES
```
or something roughly like that

Reply:
yeah, I just left that for later :)